### PR TITLE
[RTM] Makes the execute post actions compatible for the dcg

### LINKED ISSUE #91: Meta Models Filetree bei Multi-Column-Wizard
Hallo,

bei der Erstellung eines Custom Multi-Column-Elements mit Meta Models bleibt die Ausgabe im Backend leer.

Contao Version 4.4.54
Multi-Column-Wizard 3.4.7
metamodels/attribute_tablemulti Version 2.1.2

Meine Config:

```php
<?php

$GLOBALS['TL_CONFIG']['metamodelsattribute_multi']['mm_vehicle']['mc_vehicle_plans'] = array(
    'tl_class'     => 'clr',
    'minCount'     => 0,
    'maxCount'     => 20,
    'columnFields' => array(        
        'col_image' => array(
            'label'     => 'Grundriss',
            'exclude'   => true,
            'inputType' => 'fileTree',
            'eval'      => array
            (
                'fieldType'=>'radio', 
                'files'=>true, 
                'filesOnly'=>true, 
                'extensions'=>\Contao\Config::get('validImageTypes'),
                'mandatory' => true,
            ),
            'sql'		=> 'blob NULL'
        ),        
        'col_caption' => array(
            'label'     => 'Bildunterschrift',
            'exclude'   => true,
            'inputType' => 'text',
            'eval'      => array
            (
                'style'=>'width:300px',
                'maxlength'=>100, 
            )
        ),
    ),
);
``` 

Vielen Dank.

### PR DESCRIPTION
**Pull request by baumannsven**
This is the fix for can use e.g. file tree widget with the dcg.

### CODE REVIEW

**Review by zonky2 — APPROVED** [review]
works fine

**Review by discordier — APPROVED** [review]
LGTM

**Comment by baumannsven** [comment]
Travis build is failing. After the test build works fine, i switch the label wip => rtm.

**Comment by baumannsven** [comment]
@discordier I have add the two dcg packages to require-dev section. So after adds the package, the travis build crashes. I have no idea, what I can do. Can you please take a look at this?

File: .travis.yml [hunk 1]
```diff
@@ -28,20 +28,23 @@ matrix:
   allow_failures:
     - env: CONTAO_VERSION=~4.10.0
 
-before_script:
-  - echo "memory_limit=-1" >> ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/travis.ini
+sudo: false
```
Comment (discordier) [inline .travis.yml:31]: Not needed anymore. See https://docs.travis-ci.com/user/reference/overview/#deprecated-virtualization-environments

File: .travis.yml [hunk 2]
```diff
@@ -28,20 +28,23 @@ matrix:
...
+
+before_install:
+  - echo "memory_limit = -1" > travis.php.ini && phpenv config-add travis.php.ini
+
+install:
   - travis_retry composer self-update && composer --version
   - travis_retry composer require contao/core-bundle $CONTAO_VERSION --no-update
   - >
     if [ "x${TRAVIS_TAG}" != "x" ]; then
-      export COMPOSER_ROOT_VERSION=${TRAVIS_TAG}
```
Comment (discordier) [inline .travis.yml]: Please revert this, exporting and echoing below helps pin point problems as this PR is currently having.
We now have no clue at all what composer thinks it has installed as root package and therefore do not know how to make it install dcg.

File: composer.json [hunk 3]
```diff
@@ -55,12 +55,14 @@
   },
   "require": {
     "php": "^5.6 || ^7.0",
-    "contao/core-bundle": "^4.4.8",
+    "contao/core-bundle": "~4.4.0",
```
Comment (discordier) [inline composer.json]: Why the downgrade to `4.4.0`? AFAIR we upgraded to at least `4.4.8` for a reason (some bug that was incompatible with this extension).

**Review by discordier — CHANGES_REQUESTED** [review]
Please recheck `.travis.yml`.

File: composer.json (hunk 3 above)
Comment (zonky2) [reply composer.json]: https://github.com/menatwork/contao-multicolumnwizard-bundle/commit/34adec0046a3db598b9b00b6044f6d4167df9ffa#diff-d2ab9925cad7eac58e0ff4cc0d251a937ecf49e4b6bf57f8b95aab76648a9d34

File: .travis.yml (hunk 1 above)
Comment (baumannsven) [reply .travis.yml:31]: Done

File: .travis.yml (hunk 2 above)
Comment (baumannsven) [reply .travis.yml]: Done

File: composer.json (hunk 3 above)
Comment (baumannsven) [reply composer.json]: Done

File: .travis.yml (hunk 1 above)
Comment (baumannsven) [reply .travis.yml:31]: Not working. I not remove this line.

https://travis-ci.org/github/menatwork/contao-multicolumnwizard-bundle/jobs/742631080

**Comment by baumannsven** [comment]
@stefanheimes  I temporarily removed the two necessary development packages from DC General, so we called by phone.

These will be added again later.

File: .travis.yml (hunk 1 above)
Comment (discordier) [reply .travis.yml:31]: What do you mean by not working? Sudo is not needed anymore... I removed it from many `.travis.yml` files already.

Comment (baumannsven) [reply .travis.yml:31]: Oh sorry. I think do you mean the before script setting

**Pull request merged**